src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx

```diff
@@ -17,8 +17,8 @@ redirects:
 ---
 
 <Callout variant="important">
-This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA).
-</Callout>  
+This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA). If you are using the NPM package, you will need to include at least one feature.
```

Comment:
@patrickhousley  What does "include at least one feature" mean?

Reply:
When creating an instance of the agent from the NPM package, at least one feature must be included for this API to function.

Reply:
@patrickhousley Like do you mean any monitoring feature? What's an example of this?

Reply:
```
import { Agent } from '@newrelic/browser-agent/loaders/agent'
import { Metrics } from '@newrelic/browser-agent/features/metrics'
import { PageViewEvent } from '@newrelic/browser-agent/features/page_view_event'
import { PageViewTiming } from '@newrelic/browser-agent/features/page_view_timing'

const options = {
  info: { ... },
  loader_config: { ... },
  init: { ... }
}

new Agent({
  ...options,
  features: [
    // These are the features, at least one feature needs to be in this list, doesn't matter which one.
    Metrics,
    PageViewEvent,
    PageViewTiming
  ]
})
```